// Proto file representing the via layers

syntax = "proto3";

package ansys.api.edb.v1;

import "edb_messages.proto";

// Via layer service definition
service ViaLayerService {
  // Creates a via layer
  rpc Create (ViaLayerCreationMessage) returns (EDBObjMessage) {}
}

// Message for ViaLayerService::Create
message ViaLayerCreationMessage {
  string via_layer_name = 1;
  string lower_ref_layer_name = 2;
  string upper_ref_layer_name = 3;
  string material_name = 4;
}
